index.js

```diff
@@ -121,6 +128,7 @@ module.exports = class Filter extends Plugin {
       loggerName += ' > [' + annotation + ']';
     }
 
+    this.fsMerger = new FSMerger(inputTrees);
```

Comment:
by placing this on the `this` we are making `fsMerger` public, is that intentional?

Reply:
I think we can make it private.